.github/workflows/documentation-review-oauth.yml

```
@@ -0,0 +1,532 @@
+      review_type: ${{ steps.analysis.outputs.review_type }}
+      cache_key: ${{ steps.cache-key.outputs.cache_key }}
+    
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 2
+
+      - name: Skip CI Check
+        id: skip-check
+        run: |
```

actionlint (1.7.7)

"github.event.head_commit.message" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)